starter-code/src/viking.js

```diff
@@ -1,11 +1,104 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage) {
+        this.health -= damage;
+    }
+}
 
 // Viking
-class Viking {}
+let Viking = class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength)
+        this.name = name;
+    }
+    battleCry() {
+        return `Odin Owns You All!`
+    }
+    receiveDamage(damage) {
+        this.health -= damage;
```

Comment:
because it's an extend class and receiveDamage() is already declare in Soldier, you can do:
receiveDamage(damage){
  super.receiveDamage(damage)
  ...add your conditions and sentences
}